packages/percona-server-8.0-mroonga/yum/centos-6/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG FROM=centos:6
+FROM ${FROM}
+
+ARG DEBUG
+
+ENV \
+  DEVTOOLSET_VERSION=8
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
+  yum update -y ${quiet} && \
+  yum install -y ${quiet} centos-release-scl-rh && \
+  yum install -y ${quiet} \
+    https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
+    https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm \
+    epel-release && \
+  percona-release setup ps80 && \
+  yum groupinstall -y ${quiet} "Development Tools" && \
+  yum install -y ${quiet} \
+    percona-server-devel \
+    cmake28 \
+    cmake3 \
+    cyrus-sasl-devel \
+    devtoolset-${DEVTOOLSET_VERSION} \
+    gperf \
+    groonga-devel \
+    groonga-normalizer-mysql-devel \
+    intltool \
+    libaio-devel \
+    libcurl-devel \
+    libtool \
+    make \
+    ncurses-devel \
+    numactl-devel \
+    openldap-devel \
+    openssl-devel \
+    pam-devel \
+    perl-Env \
+    perl-JSON \
+    perl-Test-Simple \
+    perl-Time-HiRes \
+    pkgconfig \
+    readline-devel \
+    selinux-policy-devel \
+    sudo \
+    systemtap-sdt-devel \
+    tar \
+    time \
+    wget \
+    which \
+    zlib-devel && \
+  yum clean ${quiet} all && \
+  scl enable devtoolset-8 bash
```

Comment:
Could you use `DEVTOOLSET_VERSION` here?

Reply:
BTW, do we need this?
I think that this is automatically done by `build.sh`: https://github.com/apache/arrow/blob/master/dev/tasks/linux-packages/yum/build.sh#L110-L124